ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,93 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@st.composite
+def _conv2d_helper(draw):
+    input_dtype, \
+        x, \
+        filters, \
+        strides, \
+        dilations, \
+        data_format, \
+        padding = draw(_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC", "NCHW"]),
+        padding=st.sampled_from(["SAME", "VALID", "EXPLICIT"]),
+        type="2d",
+        dilation_min=1,
+        dilation_max=1,
+    ))
+    if padding == "EXPLICIT":
+        explicit_paddings = padding
+        pad_top = draw(st.integers(min_value=0, max_value=3))
+        pad_bottom = draw(st.integers(min_value=0, max_value=3))
+        pad_left = draw(st.integers(min_value=0, max_value=3))
+        pad_right = draw(st.integers(min_value=0, max_value=3))
+
+        if data_format == "NHWC":
+            explicit_paddings = [(0, 0),
+                                 (pad_top, pad_bottom),
+                                 (pad_left, pad_right),
+                                 (0, 0)]
+        else:
+            explicit_paddings = [(0, 0),
+                                 (0, 0),
+                                 (pad_top, pad_bottom),
+                                 (pad_left, pad_right)]
+
+    elif padding == "SAME" or padding == "VALID":
+        explicit_paddings = []
+        padding = padding
+
+    return input_dtype, \
+        x, \
+        filters, \
+        dilations, \
+        data_format, \
+        strides, \
+        padding
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Conv2D",
+    x_f_d_df=_conv2d_helper(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_Conv2D(
+    *,
+    x_f_d_df,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, \
+        x, \
+        filters, \
+        dilation, \
+        data_format, \
+        stride, \
+        padding = x_f_d_df
```

Comment:
And since `_conv2d_helper` will also be returning `explicit_paddings` you need to add it here too.